integration_tests/ipp/container-cups-tls/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM zgrab2_service_base:latest
+
+RUN apt-get update && apt-get install -y \
+  cups \
+  cups-pdf
+WORKDIR /etc/cups
+COPY cupsssl.conf cupsd.conf
+# TODO: Provide a pre-built cups-pdf.conf
+#RUN service cups restart
+
+# TODO: Actually stop service; see why this works without stopping it
+RUN service cups stop
+# TODO: Actually prevent cups from being started automatically
```

Comment:
Is cups still being started automatically despite the update-rc.d? There may be a better way to do this, but if that's the concern the TODO should probably be reworded